HACKING.rst.txt

```diff
@@ -131,10 +142,31 @@ review process.
 Other ways to contribute
 ========================
 
-There are many ways to contribute other than contributing code.  Participating
-in discussions on the scipy-user and scipy-dev *mailing lists* is a contribution
-in itself.  The `scipy.org`_ *website* contains a lot of information on the
-SciPy community and can always use a new pair of hands.
+There are many ways to contribute other than contributing code.
+
+Reviewing open pull requests (PRs) is very welcome, and a valuable way to help
+increase the speed at which the project moves forward.  Everyone who feels like
+they are competent to comment on a pull request is invited to do so; if you
```

Comment:
Some people may not feel competent enough to comment [when they are](https://en.wikipedia.org/wiki/Impostor_syndrome).  And even if there are people who are not experienced in enough to be the sole reviewer for a PR, being a co-reviewer on a PR is a good way to gain experience and become a better reviewer.  OpenDataKit is an example of a project that is [embracing that kind of approach](https://github.com/opendatakit/collect/pull/1215/files) to PR reviews.  You might consider adopting their language.